tests.py

```diff
@@ -208,8 +208,8 @@ def test_noise_sfts(self):
             SFTWindowType=self.SFTWindowType,
             SFTWindowBeta=self.SFTWindowBeta,
             sqrtSX=self.sqrtSX,
-            h0=0,
-            F0=self.F0,
+            Band=0.5,
```

Comment:
Does this solve the band estimation issue you mentioned in a comment?

Reply:
I mean this one: https://github.com/PyFstat/PyFstat/pull/207#issuecomment-705555667

Reply:
Yes. Using `F0` and `Band` takes preference over anything else.